code/modules/mob/living/carbon/xenomorph/mutators/strains/boiler/trapper.dm

```diff
@@ -1,7 +1,7 @@
 
 /datum/xeno_mutator/trapper
 	name = "STRAIN: Boiler - Trapper"
-	description = "You trade your ability to bombard, lance, and dump your acid in order to gain some speed and the ability to create acid explosions and restrain talls within them. With your longer-range vision, set up traps that immobilize your opponents and place acid mines that deal damage to talls and barricates and reduce the cooldown of your trap deployment for every tall hit. Finally, hit talls with your Acid Shotgun ability, which adds a stack of insight to empower the next trap you place once you reach a maximum of ten insight. A point-blank shot or a shot on a stunned target will instantly apply ten stacks."
+	description = "You trade your ability to bombard, lance, and dump your acid in order to gain some speed and the ability to create acid explosions and restrain talls within them. With your longer-range vision, set up traps that immobilize your opponents and place acid mines which deal damage to talls and barricates and reduce the cooldown of your trap deployment for every tall hit. Finally, hit talls with your Acid Shotgun ability which adds a stack of insight to empower the next trap you place once you reach a maximum of ten insight. A point-blank shot or a shot on a stunned target will instantly apply ten stacks."
```

Comment:
barricates????

Reply:
Thank you. I will get to that immediately!